ui/src/app/components/with-error-modal.tsx

```diff
@@ -35,14 +36,44 @@ export const withErrorModal = () => {
 
       render() {
         return <React.Fragment>
-          {this.state.show && <TextModal title={this.state.title}
+          {this.state.show && <TextModal role='alertdialog'
+                                         title={this.state.title}
                                          body={this.state.body}
                                          closeFunction={() => this.setState({show: false})}/>
           }
-          <WrappedComponent showErrorModal={(tile, body) => this.show(tile, body)}
+          <WrappedComponent showErrorModal={(title, body) => this.show(title, body)}
```

Comment:
ha

Reply:
Could this be,
```jsx
<WrappedComponent showErrorModal={this.show.bind(this)} ...>
```
to eliminate the potential for a spelling error in the first place?

Reply:
Indeed, this seems to work.